frontend/src/pages/InsideFolder.js

```diff
@@ -1,29 +1,35 @@
-import React, { useState, Component } from "react";
-import Header from "../main-components/Header";
-import Sidebar from "../main-components/Sidebar";
-import InsideFolderContent from "../main-components/InsideFolderContent";
+import React, { useState, Component, useEffect } from 'react';
+import Header from '../main-components/Header';
+import Sidebar from '../main-components/Sidebar';
+import InsideFolderContent from '../main-components/InsideFolderContent';
+import queryString from 'query-string';
 
-function InsideFolder() {
-  // Handles mobile menu button and updates sidebar view
-  const [sideSetting, setSideSetting] = useState("f");
-  const handleClick = (e) => {
-    console.log("Clicked");
-    if (sideSetting === "f") {
-      setSideSetting("t");
-    } else {
-      setSideSetting("f");
-    }
-  };
+function InsideFolder(props) {
+	const [sideSetting, setSideSetting] = useState('f');
```

Comment:
Any reason we can't use false and true?

Reply:
sideSetting is a value for a prop passed to the SideBar component. For now, it takes in 'f' or 't' in order to determine the SideBar's state.